btr-web/btr-layouts/layouts/business.vue

```diff
@@ -3,12 +3,12 @@
     <BcrosHeader />
     <BcrosBreadcrumb v-if="crumbConstructors.length > 0" :crumb-constructors="crumbConstructors" />
     <BcrosBusinessDetails />
-    <div class="mx-auto p-4 w-full max-w-bcroslg flex justify-center items-start">
-      <div class="max-w-bcrosmd p-4">
+    <div class="mx-auto px-4 w-full max-w-bcroslg flex">
+      <div>
         <slot />
       </div>
-      <div class="hidden bcroslg:block p-4 sticky top-0">
-        <BcrosWidgetsFee :fees="payFeesWidget.fees" data-cy="pay-fees-widget">
```

Comment:
updated this so that its tight against the sides and in line with the other pieces (header/breadcrumb/etc.), and also doesn't get messed up when the window is < 1360px

Reply:
I cant find which one actually had it hidden when the resolution is less then 1360px so I did it like that. But I see others do not disappear on the lower resolutions, so that makes sense not to happen here either :)